util/upload-cfn-templates.py

```diff
@@ -1,236 +0,0 @@
-#!/usr/bin/python
```

Comment:
nit: can we remove this in a separate commit and eventually PR? It doesn't have much to do with the integration testing framework right?